src/states/bootstrapping.rs

```diff
@@ -370,7 +375,7 @@ impl Display for Bootstrapping {
     }
 }
 
-#[cfg(all(test, feature = "mock"))]
+#[cfg(all(test, feature = "mock", not(feature = "mock_serialise_send")))]
```

Comment:
Can you explain why not test the bootstrapping when the mock_serialise_send feature is passed?

Reply:
It is using deserializing, and modifying types that are changed by that feature.
It is possible to make this test work, it did not seem so important, but I can fix it before we decide to merge (wanted early feedback while Viv is not on holiday).